src/commands/action/cmd-action.ts

```diff
@@ -0,0 +1,61 @@
+// https://github.com/SocketDev/socket-python-cli/blob/6d4fc56faee68d3a4764f1f80f84710635bdaf05/socketsecurity/socketcli.py
+import meowOrDie from 'meow'
+
+import { runAction } from './run-action.ts'
```

Comment:
@pvdz The `action` command is the Python-CLI port work. I don't know why it's named `action`.

Reply:
At least it's hidden.

My main questionmark is whether it is actually used or not. But us or anyone else. Since it's hidden that's less likely the case. But preferably we just drop this entirely.

It feels to me like it's some kind of GH Actions handler?

Reply:
It's hidden for now but needing more love to be exposed.